homeassistant/components/sonarr/sensor.py

```diff
@@ -267,7 +267,7 @@ def native_value(self) -> StateType:
             return len(self.data[key])
 
         if key == "queue" and self.data.get(key) is not None:
-            return self.data[key].totalRecords
+            return cast(int, self.data[key].totalRecords)
```

Comment:
I usually try to cast the least amount possible. I.e. here:

```suggestion
            return cast(SonarrQueue, self.data[key]).totalRecords
```